From e329e9ec19db0e940a4f7a8959a7a40b692413d4 Mon Sep 17 00:00:00 2001
From: "Pawel Hajdan, Jr" <phajdan.jr@chromium.org>
Date: Wed, 31 May 2017 11:06:38 +0000
Subject: [PATCH] Fix GN bootstrap

TBR=brettw

BUG=none

Review-Url: https://codereview.chromium.org/2913973002 .
Cr-Commit-Position: refs/heads/master@{#475873}
---
 tools/gn/bootstrap/bootstrap.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 6f2f5b1..0b03d26 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -487,6 +487,7 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/sys_info.cc',
       'base/task_runner.cc',
       'base/task_scheduler/delayed_task_manager.cc',
+      'base/task_scheduler/environment_config.cc',
       'base/task_scheduler/post_task.cc',
       'base/task_scheduler/priority_queue.cc',
       'base/task_scheduler/scheduler_lock_impl.cc',
-- 
2.9.4
